tests/main/enable-disable/task.yaml

```diff
@@ -0,0 +1,21 @@
+summary: Check that enable/disable works
+
+execute: |
+ echo "Install hello-world and ensure it runs"
```

Comment:
While we're here, can you fix the indentation after the new 4 spaces convention?